codespell_lib/data/dictionary.txt

```diff
@@ -29714,6 +29737,7 @@ utilies->utilities
 utililties->utilities
 utilis->utilise
 utilites->utilities
+utilitizes->utilizes
```

Comment:
You could suggest the singular and potentially lots of other versions.

Reply:
added a few

Reply:
Would you mind adding the -ise versions of these too please?

Reply:
yup, done

Reply:
Thanks.